source/includes/cross-version-sync.rst

```diff
@@ -0,0 +1,3 @@
+Starting in 1.7.0, ``mongosync`` can synchronize data between clusters
+running different MongoDB server versions. For example, synchronize
+clusters running MongoDB 6.0 and 7.0.
```

Comment:
1.6 introduced support for different patch releases, (e.g., 6.0.8 source to 6.0.9).  It might help to stress that this involves synchronization between major versions.

Reply:
Will do.